2015-06-25 07:34:00 Error: [Cake\Routing\Exception\MissingControllerException] Controller class User could not be found.
Exception Attributes: array (
  'class' => 'User',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /github/alumni.upj.ac.id/user
Stack Trace:
#0 C:\xampp\htdocs\github\alumni.upj.ac.id\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-25 08:01:08 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Index could not be found.
Exception Attributes: array (
  'class' => 'Index',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /github/alumni.upj.ac.id/index
Stack Trace:
#0 C:\xampp\htdocs\github\alumni.upj.ac.id\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-01 04:11:23 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Img could not be found.
Exception Attributes: array (
  'class' => 'Img',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /alumni/img/logo2
Stack Trace:
#0 C:\xampp\htdocs\alumni\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-01 04:11:24 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Img could not be found.
Exception Attributes: array (
  'class' => 'Img',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /alumni/img/logo2
Stack Trace:
#0 C:\xampp\htdocs\alumni\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-03 03:57:10 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\alumni\src\Controller\AlumniController.php, line 23]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-03 03:57:10 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /alumni/alumni
Stack Trace:
#0 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 23)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-03 03:57:16 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Login could not be found.
Exception Attributes: array (
  'class' => 'Login',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /alumni/login
Stack Trace:
#0 C:\xampp\htdocs\alumni\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-03 03:57:56 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\alumni\src\Controller\AlumniController.php, line 23]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-03 03:57:56 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /alumni/alumni
Stack Trace:
#0 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 23)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-03 03:58:12 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\alumni\src\Controller\AlumniController.php, line 23]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-03 03:58:12 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /alumni/alumni/add
Stack Trace:
#0 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 23)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-03 04:00:01 Error: Fatal Error (4): syntax error, unexpected '->' (T_OBJECT_OPERATOR) in [C:\xampp\htdocs\alumni\src\Model\Table\AlumniTable.php, line 78]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-03 04:00:01 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '->' (T_OBJECT_OPERATOR)
Request URL: /alumni/alumni/add
Stack Trace:
#0 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 78)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-03 04:00:08 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\alumni\src\Controller\AlumniController.php, line 23]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-03 04:00:08 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /alumni/alumni/add
Stack Trace:
#0 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 23)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-03 04:00:13 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\alumni\src\Controller\AlumniController.php, line 23]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-03 04:00:13 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /alumni/alumni/
Stack Trace:
#0 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 23)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-03 04:00:17 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Alumnus could not be found.
Exception Attributes: array (
  'class' => 'Alumnus',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /alumni/alumnus/
Stack Trace:
#0 C:\xampp\htdocs\alumni\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-03 04:00:24 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\alumni\src\Controller\AlumniController.php, line 23]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-03 04:00:24 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /alumni/alumni/
Stack Trace:
#0 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 23)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-03 05:59:59 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\alumni\src\Controller\AlumniController.php, line 23]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-03 05:59:59 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /alumni/alumni
Stack Trace:
#0 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 23)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-03 06:21:27 Error: [Cake\Controller\Exception\MissingComponentException] Component class Component could not be found.
Exception Attributes: array (
  'class' => 'Component',
  'plugin' => NULL,
)
Request URL: /alumni/berita
Stack Trace:
#0 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Core\ObjectRegistry.php(87): Cake\Controller\ComponentRegistry->_throwMissingClassError('', NULL)
#1 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Controller\Controller.php(329): Cake\Core\ObjectRegistry->load('', Array)
#2 C:\xampp\htdocs\alumni\src\Controller\AppController.php(40): Cake\Controller\Controller->loadComponent('')
#3 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Controller\Controller.php(278): App\Controller\AppController->initialize()
#4 [internal function]: Cake\Controller\Controller->__construct(Object(Cake\Network\Request), Object(Cake\Network\Response), 'Berita')
#5 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Routing\Filter\ControllerFactoryFilter.php(87): ReflectionClass->newInstance(Object(Cake\Network\Request), Object(Cake\Network\Response), 'Berita')
#6 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Routing\Filter\ControllerFactoryFilter.php(49): Cake\Routing\Filter\ControllerFactoryFilter->_getController(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Routing\DispatcherFilter.php(145): Cake\Routing\Filter\ControllerFactoryFilter->beforeDispatch(Object(Cake\Event\Event))
#8 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Event\EventManager.php(389): Cake\Routing\DispatcherFilter->handle(Object(Cake\Event\Event), Object(Cake\Network\Request), Object(Cake\Network\Response))
#9 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#10 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Event\EventManagerTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#11 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(62): Cake\Routing\Dispatcher->dispatchEvent('Dispatcher.befo...', Array)
#12 C:\xampp\htdocs\alumni\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#13 {main}


2015-07-03 06:23:42 Error: [Cake\Network\Exception\NotFoundException] Not Found
Request URL: /alumni/
Stack Trace:
#0 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\View\View.php(833): include()
#1 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\View\View.php(793): Cake\View\View->_evaluate('C:\\xampp\\htdocs...', Array)
#2 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\View\View.php(464): Cake\View\View->_render('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Controller\Controller.php(582): Cake\View\View->render('home', NULL)
#4 C:\xampp\htdocs\alumni\src\Controller\PagesController.php(57): Cake\Controller\Controller->render('home')
#5 [internal function]: App\Controller\PagesController->display('home')
#6 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Controller\Controller.php(411): call_user_func_array(Array, Array)
#7 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#8 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PagesController))
#9 C:\xampp\htdocs\alumni\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#10 {main}


2015-07-03 06:23:50 Error: [Cake\Network\Exception\NotFoundException] Not Found
Request URL: /alumni/
Stack Trace:
#0 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\View\View.php(833): include()
#1 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\View\View.php(793): Cake\View\View->_evaluate('C:\\xampp\\htdocs...', Array)
#2 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\View\View.php(464): Cake\View\View->_render('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Controller\Controller.php(582): Cake\View\View->render('home', NULL)
#4 C:\xampp\htdocs\alumni\src\Controller\PagesController.php(57): Cake\Controller\Controller->render('home')
#5 [internal function]: App\Controller\PagesController->display('home')
#6 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Controller\Controller.php(411): call_user_func_array(Array, Array)
#7 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#8 C:\xampp\htdocs\alumni\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PagesController))
#9 C:\xampp\htdocs\alumni\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#10 {main}


